apps/omg/lib/omg/state/transaction/validator.ex

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I started this because UTXO management will likely change and I want this to look more _deterministic_.

Reply:
So this module is reached only for Transaction processing in _Core.exec_.